### ISSUE TITLE
Don't bring the app to the front after refreshing the index

### ISSUE THREAD

**Issue by jpschewe** [laullon/gitx#159]
I noticed that gitx refreshes the index each time it gains focus. This can make the app a little slow to switch between. However what is really annoying is that after the refresh has finished the app brings itself to the front and steals focus from whatever app currently has focus. So if I switch to gix and then off to something else, gitx will then steal the focus back once the refresh finishes.

**Comment by laullon** [comment]
disable auto reload on preferences

**Comment by jpschewe** [comment]
Yes, I've seen that solution. I really don't think that's much of a solution. An application stealing focus is just a bad behavior.

**Comment by laullon** [comment]
which version are you using?

**Comment by jpschewe** [comment]
Latest stable - 0.8.4

**Comment by williscool** [comment]
has there been any movement on this?

**Comment by jphalip** [comment]
I believe this has been fixed as a side effect of the recent commits.
